base/shell/explorer/lang/en-US.rc

```diff
@@ -28,6 +28,19 @@ BEGIN
     END
 END
 
+IDM_TASKGRP MENUEX
```

Comment:
What about all the other languages? They don't get the menu?

Reply:
Now they are added but translation is left

Reply:
It's OK :)